client/src/legacy/entwine/LeftAndMain_Subsites.js

```diff
@@ -0,0 +1,190 @@
+/* jslint browser: true, nomen: true */
+/* global window, jQuery */
+import React from 'react';
+import ReactDom from 'react-dom';
+import { loadComponent } from 'lib/Injector';
+import createEvent from 'legacy/createEvent';
+import changeActiveSubsite from 'lib/changeActiveSubsite';
+
+jQuery.entwine('ss', ($) => {
+  $('#SubsitesSelect').entwine({
+    ModalNode: null,
+    onadd() {
+      // Storage has updated - another tab has changed the subsite info
+      window.addEventListener('storage', (storageEvent) => {
```

Comment:
`storage` is possibly a bit too generic', `subsitestorage` is more likely to never have a conflict

Reply:
Yeah that's the actual api name of the event type we're using.
https://developer.mozilla.org/en-US/docs/Web/API/StorageEvent

Reply:
Indeed, this is not determined by us. It must be 'storage'.